apps/angular/projection/src/app/ui/card/card.component.html

```diff
@@ -1,27 +1,19 @@
 <div
   class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
   [class]="customClass">
-  <img
-    *ngIf="type === CardType.TEACHER"
-    src="assets/img/teacher.png"
-    width="200px" />
-  <img
-    *ngIf="type === CardType.STUDENT"
-    src="assets/img/student.webp"
-    width="200px" />
+  <ng-content select="[image]"></ng-content>
```

Comment:
nit: you can also select the tag directly: `select=img`: this way you don't have to create a div. Or you can add the image selector directly on the img DOM element

Reply:
I fix that. thanks for comments sir.